projects/novo-elements/src/elements/avatar/Avatar.scss

```diff
@@ -0,0 +1,20 @@
+:host {
+  img {
+    width: 30px;
+    height: 30px;
+    border-radius: 2em;
+  }
+  // @each $basic, $color in $all-colors {
```

Comment:
I'm not a fan of commented out code. If there's a reason we want code to be commented out, then please add a comment at the top stating why the commented out code is there.

Reply:
copy pasta from the original component.